spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/catalog/GravitinoCatalogManager.java

```diff
@@ -40,9 +41,13 @@ private GravitinoCatalogManager(String gravitinoUri, String metalakeName) {
   }
 
   public static GravitinoCatalogManager create(String gravitinoUrl, String metalakeName) {
-    Preconditions.checkState(
```

Comment:
With `Preconditions`, `SparkIcebergCatalogIT` always fails when it initializes the sparkSession instance. 
Because `gravitinoCatalogManager` is a static variable, `gravitinoCatalogManager` must not be null after `SparkHiveCatalogIT` is executed. When `SparkIcebergCatalogIT` is initialized, Instantiating the `gravitinoCatalogManager` again will fail.

Reply:
I think the proper way is to set global gravitinoCatalogManager to null when close (and must check the global instance is the same instance that to be closed). when create a `gravitinoCatalogManager` it's useful to check the global state.

Reply:
fixed